src/test/projects/class/detectsApiChangesInExportedSuperClassInProtectedInnerClass/new/pom.xml

```diff
@@ -10,7 +10,7 @@
     </parent>
 
     <artifactId>empty-project</artifactId>
-    <version>1.0.1-SNAPSHOT</version>
+    <version>1.0.1</version>
```

Comment:
There are other test artifacts that use 1.0.0-SNAPSHOT. You will find one per test project inside the "old" folder